Translation

MasterCyria opened the pull request.

Crevox commented:
Whoa someone is helping out!

Looks good, just for the first line make sure the capitalization falls in line with other things we've done.

Also, in general we translate "Rim" as "Rift" to maintain consistency with Dark Arisen. I think we have "Rim Stone" translated elsewhere, so be sure it's consistent with other places we've done.

Also, make sure none of those white space changes affect the file format.

MasterCyria commented:
Glad to help! I firmly believe that having a more fully flushed-out English patch will draw more interest for English speaking languages, which will drum more interest to Capcom bringing it over here.

Changed Rim -> Rift. If I run across that elsewhere, I'll change it.

Corrected the white-space issues. I didn't mean to do that :)

When you say for the first line with the capitalization, what exactly are you referring to?
